app/api/attest/route.ts

```diff
@@ -37,36 +34,10 @@ export async function POST(req: NextRequest) {
     return NextResponse.json({ error: 'User is not diamond hands' }, { status: 400 })
   }
 
-  const provider = new JsonRpcProvider(process.env.RPC_PROVIDER)
-  const signer = new Wallet(PRIVATE_KEY, provider);
-
-  const proxy = new EIP712Proxy(PROXY_CONTRACT_ADDRESS, { signer: signer })
-
-  const delegated = await proxy.getDelegated()
-
-  const attestationType = ATTESTATION_CONFIG['diamond-hand'];
-
-  const params = {
-    schema: attestationType.schemaUID,
-    recipient: walletAddress,
-    expirationTime: 0n,
-    revocable: false,
-    refUID: '0x0000000000000000000000000000000000000000000000000000000000000000',
-    data: attestationType.encoder.encodeData([{
-      name: 'hasDiamondHand',
-      type: 'bool',
-      value: true
-    }]),
-    value: 0n,
-    deadline: 0n
-  };
-
-  const response = await delegated.signDelegatedProxyAttestation(params, signer);
-  const signedResponse = jsonStringifyBigInt({
-    message: response.message,
-    signature: response.signature,
-  })
-
+  if (await hasDiamondHand(walletAddress)) {
```

Comment:
I suggest changing this function name to `attestedDiamondHand`, as it is easy to confuse with the existing `isDiamondHand` (which is just checking if the wallet is whitelisted)